text/0148-sampling-probability.md

```diff
@@ -0,0 +1,744 @@
+inclusion probability.  Inclusion probability in the context of
+tracing is the *effective* probability of the Sampler returning
+`RECORD_AND_SAMPLE` when invoked, which can be decided locally or 
+derived from the context when the [W3C Trace Context is-sampled
+flag](https://www.w3.org/TR/trace-context/#sampled-flag) is in use.
+
+#### Adjusted count span attribute
+
+The recommended way to convey inclusion probability *for events*
+sampled in OpenTelemetry is in the form of an **adjusted count**,
+which is the reciprocal (i.e., mathematical inverse function) of
+inclusion probability.
+
+The implied goal of probability sampling is to support estimating the
+count of spans in the population using the spans that were sampled.
+The adjusted count associated with a Span is the expected value of the
+number of identical Spans within the population that each Span
+represents.  Probability samplers SHOULD ensure the samples they
+compute are unbiased, which implies that the expected value of the sum
+of adjusted counts in the sample equals the true count of spans in the
+population.
+
+Attributes used to express the adjusted count in an unbiased
+probability sampling scheme SHOULD use a Span attribute name with
+prefix `sampling.` and with suffix `.adjusted_count` (e.g.,
+`sampling.sampler_name.adjusted_count`).  Adjusted count attributes
+MAY be integer or floating-point values.
+
+#### Inclusion probability tracestate value
+
+The recommended way to convey sampling probability *for contexts* in
+OpenTelemetry is through the W3C Trace Context tracestate using the
+key `head_probability`.
+
+Probability Samplers SHOULD encode the effective sampling inclusion
+probability using tracestate, for the context that was in effect when
+the W3C is-sampled bit was set.  The tracestate field SHOULD be set in
+both sampled and unsampled cases, to convey the inclusion probability
+even for unsampled contexts.
+
+The `head_probability` tracestate key is set to a floating-point
+number greater than or equal to 0 and less than or equal to 1.  The
+floating point precision of the number SHOULD follow implementation
+language standards and SHOULD be high enough to identify when Samplers
+have different inclusion probabilities.
+
+#### Counting probabilistically sampled spans
+
+Consumers of a stream of span data that may or may not have been
+sampled can follow these steps to count or approximately count the
+total number of spans in the population.
+
+For each span processed, locate all span attributes with prefix
+`sampling.` and suffix `.adjusted_count`.  If there are no adjusted
+count attributes on the span, count a single event.  If there is
+exactly one adjusted count attribute, count that many identical span
+events.
+
+If there are more than one adjusted count attribute on the span, the
+processor SHOULD make a consistent choice for spans of a given
+resource.  By default, the SDK's first preference MUST the builtin
```

Comment:
a) why this specific priority?
b) this section describes downstream processor, why the reference to SDK?

Reply:
I threw this paragraph away and have removed all references to multiple adjusted counts.

I was trying to solve a problem when more than one framing is being used, but I see this was a mistake.

The key property is that if we expect to count all spans, we want to ensure the framing refers to all spans, so I've written that `sampling.adjusted_count` SHOULD refer to the whole population of spans.

Reply:
still have the same two questions

Reply:
I meant to remove this section, it's now removed.